lib/pengine/tests/native/native_find_rsc_test.c

```diff
@@ -0,0 +1,683 @@
+/*
+ * Copyright 2022 the Pacemaker project contributors
+ *
+ * The version control history for this file may have further details.
+ *
+ * This source code is licensed under the GNU Lesser General Public License
+ * version 2.1 or later (LGPLv2.1+) WITHOUT ANY WARRANTY.
+ */
+
+#include <crm_internal.h>
+
+#include <stdarg.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <setjmp.h>
+#include <cmocka.h>
+
+#include <crm/common/xml.h>
+#include <crm/pengine/internal.h>
+#include <crm/pengine/status.h>
+#include <crm/pengine/pe_types.h>
+
+/* Needed to access replicas inside a bundle. */
+#define PE__VARIANT_BUNDLE 1
+#include <lib/pengine/variant.h>
```

Comment:
Unrelated to this PR, just FYI, my long-term plan is to add enough (internal-only) accessors that the variant.h stuff can be local to each variant .c file.

Reply:
That would be really nice to have.  It comes up for me every now and then, and having to do this define/include trick feels dirty.